samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -251,8 +278,49 @@ static int lc3_enable(struct broadcast_sink_stream *sink_stream)
 		return ret;
 	}
 
-	frames_per_sdu = bt_audio_codec_cfg_get_frame_blocks_per_sdu(sink_stream->stream.codec_cfg,
-								     true);
+	ret = bt_audio_codec_cfg_get_chan_allocation(sink_stream->stream.codec_cfg,
+						     &sink_stream->chan_allocation);
+	if (ret != 0) {
+		printk("Error: Channel allocation not set, invalid configuration for LC3");
+		return ret;
+	}
+
+	ret = bt_audio_codec_cfg_get_octets_per_frame(sink_stream->stream.codec_cfg);
+	if (ret > 0) {
+		sink_stream->lc3_octets_per_frame = (uint16_t)ret;
+	} else {
+		printk("Error: Octets per frame not set, invalid configuration for LC3");
+		return ret;
+	}
+
+	ret = bt_audio_codec_cfg_get_frame_blocks_per_sdu(sink_stream->stream.codec_cfg, true);
+	if (ret > 0) {
+		sink_stream->lc3_frames_blocks_per_sdu = (uint8_t)ret;
+	} else {
+		printk("Error: Frame blocks per SDU not set, invalid configuration for LC3");
+		return ret;
+	}
+
+	/* An SDU can consist of X frame blocks, each with Y frames (one per channel) of size Z in
+	 * them. The minimum SDU size required for this is X * Y * Z.
+	 */
+	chan_alloc_bit_cnt = get_chan_cnt(sink_stream->chan_allocation);
+	min_sdu_size_required = chan_alloc_bit_cnt * sink_stream->lc3_octets_per_frame *
+				sink_stream->lc3_frames_blocks_per_sdu;
+	if (min_sdu_size_required < sink_stream->stream.qos->sdu) {
```

Comment:
it will be less confusion if renaming min_sdu_size_required to be max_sdu_size_required.

 just looking at code I will think it is ok stream's sdu size is bigger than min_sdu_size_required

Reply:
I will rename to `sdu_size_required` to avoid `min`/`max` in the name (hopefully reducing the confusion